nav2_system_tests/src/multi_robot/test_multi_robot_node.py

```diff
@@ -0,0 +1,213 @@
+#! /usr/bin/env python3
```

Comment:
We should factor out the common code between this and test_system_node.py. Maybe the test can take the number of robots as a parameter and adjust accordingly?

Reply:
Agree, I was considering that as future work but can propose something and push it as a separate commit on this PR to review.